features/lorawan/LoRaWANStack.cpp

```diff
@@ -96,19 +79,55 @@ LoRaWANStack::LoRaWANStack()
 
      memset(&_lw_session, 0, sizeof(_lw_session));
      memset(&_rx_msg, 0, sizeof(_rx_msg));
-
-     LoRaMacPrimitives.mcps_confirm     = callback(this, &LoRaWANStack::mcps_confirm_handler);
-     LoRaMacPrimitives.mcps_indication  = callback(this, &LoRaWANStack::mcps_indication_handler);
-     LoRaMacPrimitives.mlme_confirm     = callback(this, &LoRaWANStack::mlme_confirm_handler);
-     LoRaMacPrimitives.mlme_indication  = callback(this, &LoRaWANStack::mlme_indication_handler);
 }
```

Comment:
Add initialization of _tx_msg and _rx_payload.

Reply:
Added in constructor.